hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/ECBlockOutputStream.java

```diff
@@ -80,6 +80,10 @@ public void write(byte[] b, int off, int len) throws IOException {
         writeChunkToContainer(ChunkBuffer.wrap(ByteBuffer.wrap(b, off, len)));
   }
 
+  public void write(ByteBuffer buff) throws IOException {
+    this.currentChunkRspFuture = writeChunkToContainer(ChunkBuffer.wrap(buff));
```

Comment:
Nit: should it return the future and let coordinator use it directly instead of `getCurrent...`?

Reply:
sure